compiler+runtime/bin/jank/compiler+runtime/core.clj

```diff
@@ -23,7 +23,7 @@
                                            :coverage (util/get-env "JANK_COVERAGE" "off")})
 
   ; Bash tests
-  #_(jank.compiler+runtime.bash-test/-main {:enabled? build?})
+  (jank.compiler+runtime.bash-test/-main {:enabled? build?})
```

Comment:
CI in this PR is failing, from what I can tell. Is that expected? If so, we need to leave these disabled.

Reply:
Disabled these tests as the binary can't find the precompiled headers yet.